### PR TITLE
contrib/{sdl12-compat,sdl1.2_image,sdl1.2_mixer}: new package

### PR DESCRIPTION
**Pull request by z-erica**

### CODE REVIEW

File: contrib/sdl1.2_mixer/template.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+build_style = "gnu_configure"
+# make sure they're pulled as runtime deps
+configure_args = [
+    "--disable-music-mod-shared",
+    "--disable-music-ogg-shared",
+    "--disable-music-flac-shared",
+]
+configure_gen = []
+make_cmd = "gmake"
+hostmakedepends = ["automake", "gmake", "pkgconf"]
```
Comment (triallax) [inline contrib/sdl1.2_mixer/template.py:13]: also noticed that configure complains about missing /usr/bin/file but not sure if that's important, might as well add file to hostdeps anyway
```suggestion
hostmakedepends = ["file", "gmake", "pkgconf"]
```

Comment (nekopsykose) [reply contrib/sdl1.2_mixer/template.py:13]: it's literally not used for anything

File: contrib/sdl1.2_mixer/template.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+pkgver = "1.2.12"
+pkgrel = 0
+build_style = "gnu_configure"
+# make sure they're pulled as runtime deps
+configure_args = [
+    "--disable-music-mod-shared",
+    "--disable-music-ogg-shared",
+    "--disable-music-flac-shared",
+]
+configure_gen = []
```
Comment (nekopsykose) [inline contrib/sdl1.2_mixer/template.py:11]:
```suggestion
configure_gen = ["./autogen.sh"]
```

File: contrib/sdl1.2_mixer/template.py (hunk 1 above)
Comment (triallax) [reply contrib/sdl1.2_mixer/template.py:13]: /shrug, it does try to run it and that fails and complains but ok

**Pull request merged**